lib/cisco_node_utils/feature.rb

```diff
@@ -23,6 +23,25 @@ class Feature < NodeUtil
     # however, for test purposes it is sometimes convenient to support
     # feature disablement for cleanup purposes.
     # ---------------------------
+    def self.bfd_enable
+      return if bfd_enabled?
+      config_set('feature', 'bfd', state: '')
+    end
+
+    def self.bfd_disable
+      return unless bfd_enabled?
+      config_set('feature', 'bfd', state: 'no')
```

Comment:
I checked this and it is not really taking 25 seconds. The message says it coul;d take upto 25 sec but in minitest everything seems fast. So I did not add any code. For the method, why it is needed at all, I thought that this may be needed in the destroy method of node utils. I was looking at the fabricpath_gobal.rb
https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/fabricpath_global.rb#L101

If I get rid of this disabled method, then do we need destroy method at all? Let me know.